integreat_cms/cms/views/events/event_list_view.py

```diff
@@ -202,6 +204,16 @@ def get(self, request, *args, **kwargs):
         )
         chunk = request.GET.get("page")
         event_chunk = paginator.get_page(chunk)
+
+        # DeepL available
+
+        if settings.DEEPL_ENABLED:
+            deepl = DeepLApi()
+            DEEPL_AVAILABLE = deepl.check_availability(request, language_slug)
+        else:
+            messages.error(request, _("Automatic translations are disabled"))
```

Comment:
```suggestion
```
I don't think it makes sense to permanently show an error message on the list view when DeepL is disabled. When it's disabled, the user should not notice that we implemented this feature at all.